migration/run.spec.ts

```diff
@@ -65,6 +65,11 @@ describe('Migration script', () => {
       expect(result).not.toContain('ngOnDestroy()');
     });
 
+    it('should place @UntilDestroy decorator to a new line after the comment', () => {
```

Comment:
Please add a test case for an existing class with multiple decorators.

Reply:
@NetanelBasal what do you mean? `\n@UntilDestroy\n@Component`?